tests/statespace_testfixture.h

```diff
@@ -351,6 +351,35 @@ R"(20
 30 y_1_2 19
 )";
 
+template <typename StateSpace>
+void TestNormSmall() {
```

Comment:
Does this test differentiate between the old and new behavior? i.e. will it fail if run without the other changes in this PR?

Reply:
Yes, it will fail without the other changes in this PR.